README.md

```diff
@@ -1,29 +1,29 @@
 ## Headerless.org
 
-The Headerless Organisation committed itself to promote a streamlined license and copyright process in FOSS projects.
+The **Headerless Initiative** is committed to promote a streamlined license and copyright process in FOSS projects.
```

Comment:
I would like to keep the "Organisation" reference in somehow. I've therefore adapted it to `The **Headerless.org Initiative** is committed`